### DISCUSSION TITLE
Updating firmware miyoo mini+ i have no wifi

### DISCUSSION THREAD

**Discussion by Greentrail603** [OnionUI/Onion#832] [Q&A] [answered]
got my miyoo mini plus. got a pre installed onion os sd card. no idea what the root is as im not as tech savvy i did not set this card up myself so i guess shame on me. i just need to update so i have wifi i currently have no icon or access on my miyoo mini plus. some help would be great. i tried downloading latest version but following instructions i feel my device does not update and nothing is accomplished. would be easier to update through the wifi but of course thats what i am lacking

**Comment by chaiyaph0n** [comment]
I got the same issue too, what I have done is I got the stock os pre-installed on the SD card, and I update the firmware from [lomiyoo MINI+PLUS 0505 firmware](https://www.lomiyoo.com/jiaocheng.html), then install Onion V4.2.0.1 beta, probably the root cause is it's a beta version not sure about this, no wifi icon shown in setting but if I remove the SD card and turn on the device with stock os, I can see the wifi icon there but it can't find any wifi nearby

**Reply by chaiyaph0n** [reply, marked as answer]
This issue resolves after install 4.2.0 beta - [Onion-v4.2.0-beta-dev-b321c9a.zip](https://github.com/OnionUI/Onion/releases/download/latest/Onion-v4.2.0-beta-dev-b321c9a.zip)

**Reply by hussyfelix** [reply]
hello, i got the same issue. as you mentioned before This Issue resolves after install 4.2.0 beta. did you reinstall everything include Miyo firmware and Onion os ?
or you just direct update the onion os 4.20 ?

**Reply by hussyfelix** [reply]
i did both, reinstalled the Miyo Mini Plus OS and reinstalled Onion OS but still no available wifi network. the wifi keep scanning for no result. Currently i’m using Miyo firmware 202306282128 and Onion OS https://github.com/OnionUI/Onion/releases/download/latest/Onion-v4.2.0-RC-snapshot-a5064207.zip
please help if you got any solutions.

**Reply by Aemiii91** [reply]
Try Apps -> Tweaks -> Advanced -> Reset settings -> Reset MainUI settings

**Reply by hussyfelix** [reply]
i’ve tried. still no luck.

**Reply by XK9274** [reply]
Do you have the 8188fu.ko file in your SD root? Wifi will continuously scan if this file is missing/corrupt or can't load in for some reason.

**Reply by hussyfelix** [reply]
yes. ive replaced the 8188fu.ko from the stock SD card. and still scan nothing.

**Reply by pixfierro** [reply]
> i’ve tried. still no luck.

Have you solved it? I have the same problem...
